lib/glific/flows.ex

```diff
@@ -87,7 +87,8 @@ defmodule Glific.Flows do
       {:ok, _} =
         FlowRevision.create_flow_revision(%{
           definition: FlowRevision.default_definition(flow),
-          flow_id: flow.id
+          flow_id: flow.id,
+          status: "done"
```

Comment:
Default status should be draft right?

Reply:
We might need to decide later, whether a flow should have at least one revision with status done.